openerp/addons/base/migrations/9.0.1.3/pre-migration.py

```diff
@@ -59,6 +59,7 @@ def cleanup_modules(cr):
         cr, [
             ('contacts', 'mail'),
             ('marketing_crm', 'crm'),
+            ('email_template', 'mail'),  # mail_template class
```

Comment:
No, that should be in the corresponding module. This is only the initiation of the migration, for setting which modules to upgrade. When upgrading the mail module is when renamings should happen.